Sources/QuranAudioKit/Reciter/DownloadedRecitersService.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  DownloadedRecitersService.swift
+//
+//
+//  Created by Zubair Khan on 12/31/22.
+//
+
+import Foundation
+
+public class DownloadedRecitersService {
+
+    public init() {
+    }
+
+    public func downloadedReciters(_ allReciters: [Reciter], _ recentReciters: [Reciter]) -> [Reciter] {
```

Comment:
Let's separate downloaded reciters from recent reciters logic. I would expect DownloadedRecitersService to return all downloaded reciters without any filtering logic for recent reciters. Then the UI/ViewModel/UseCase is the one that does the logic that filter our recent reciters from the downloaded reciters since this is just a UI logic not business logic.